### PR TITLE
Code Coverage

### PR DESCRIPTION
**Pull request by 7h3kk1d**
Code coverage screenshot of Elaborator module

- [x] Add documentation
- [x] Add `make coverage` to make file 
- [ ] Integrate with CI somehow. (Maybe https://about.codecov.io/)

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -56,3 +56,63 @@ setup.log
 
 # Backup of opam lock file
 hazel.opam.locked.old
+_coverage/coverage.css
```
Comment (7h3kk1d) [inline .gitignore]: Why would vscode do this

**Comment by 7h3kk1d** [comment]
Marking this ready for review as it's done but can't go in before https://github.com/hazelgrove/hazel/pull/1341

**Review by cyrus- — COMMENTED** [review]
check to make sure this isn't instrumenting when not running the test build target, otherwise looks gtg

**Comment by 7h3kk1d** [comment]
> check to make sure this isn't instrumenting when not running the test build target, otherwise looks gtg

@cyrus- Yeah, based on my reading it only gets enabled when instrumentation is enabled in the dune command which I'm only doing in make coverage

**Review by cyrus- — APPROVED** [review]

**Pull request merged**